README.MD

```diff
@@ -6,11 +11,72 @@
 
 ## How to contribute:
 
-* adding keywords: submit a PR to [scaladex-metadata](https://github.com/scalacenter/scaladex-metadata)
-* claiming you project: submit a PR to [scaladex-data](https://github.com/scalacenter/scaladex-data)
-* improving the index: join the [gitter.im channel](https://gitter.im/scalacenter/scaladex)
+* [Deprecated](#How-to-publish-on-scaladex) adding keywords: submit a PR to [scaladex-metadata]
+* [Deprecated](#How-to-publish-on-scaladex) claiming you project: submit a PR to [scaladex-data]
+* improving the index: join the [gitter.im channel]
 
-## How to run:
+## Hot to publish on scaladex
```

Comment:
@heathermiller @MasseGuillaume Could one of you maybe check the description of errors / miss understanding / mistakes.